chain/ethereum.ts

```diff
@@ -33,8 +35,22 @@ export namespace ethereum {
    * A dynamically typed value used when accessing Ethereum data.
    */
   export class Value {
-    kind: ValueKind
-    data: ValuePayload
+    constructor(
+      public kind: ValueKind,
+      public data: ValuePayload,
+    ) {}
+
+    @operator('<')
+    lt(other: Value): boolean {
+      assert(false, "Less than operator isn't supported in Value")
+      return false
+    }
+
+    @operator('>')
+    gt(other: Value): boolean {
+      assert(false, "Greater than operator isn't supported in Value")
```

Comment:
`assert(false, "foo")` is just `abort("foo")`.

Reply:
Fixed!